src/osip-simulation-view/src/main/java/edu/kit/pse/osip/simulation/view/dialogs/AboutDialog.java

```diff
@@ -1,7 +1,137 @@
+    private static final String PROG_NAME = "OSIP";
+    
+    /**
+     * Constructor of AboutDialog
+     */
+    public AboutDialog() {
+        configure();
+    }
+
+    private void setDialogSize() {
+        this.setMinWidth(MIN_WINDOW_WIDTH);
+        this.setMinHeight(MIN_WINDOW_HEIGHT);
+    }
+
+    private void configureIcon(Image image, ImageView icon) {
+        icon.setImage(image);
+        icon.setFitWidth(8 * FONT_SIZE);
+        icon.setPreserveRatio(true);
+        icon.setSmooth(true);
+    }
+    
+    private void configureScrollPane(ScrollPane scrollPane, TextFlow textFlow) {
+        scrollPane.setFitToWidth(true);
+        scrollPane.setContent(textFlow);
+        scrollPane.setPadding(new Insets(ViewConstants.ELEMENTS_GAP));
+    }
+
+    private Text getText(String id, Font font) {
+        Text text = new Text();
+        text.setText(Translator.getInstance().getString(id) + "\n");
+        if (font != null) {
+            text.setFont(font);
+        }
+        return text;
+    }
+    
+    private Text newLine() {
+        return new Text("\n");
+    }
+    
+    private Text getLicenseText(String path) {
+        InputStreamReader reader = null;
+        reader = new InputStreamReader(getClass().getClassLoader().getResourceAsStream(path));        
+        String fileContents = "";      
+        int i;       
+        try {
+            while ((i = reader.read()) != -1) {
+                char ch = (char) i;      
+                fileContents = fileContents + ch; 
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        Text output = new Text(fileContents);
+        output.setFont(Font.font("Monospaced", FONT_SIZE));
+        return output;
+    }
+    
+    private void configure() {
+        this.setTitle(Translator.getInstance().getString("monitoring.aboutdialog.title"));
+        Image imageIcon = new Image(getClass().getClassLoader().getResourceAsStream("icon.png"));
+        this.getIcons().add(imageIcon);
+        GridPane grid = new GridPane();
+        grid.setPadding(new Insets(ViewConstants.ELEMENTS_GAP));
+        
+        HBox hbox = new HBox();
+        hbox.setPadding(new Insets(ViewConstants.ELEMENTS_GAP));
+        hbox.setSpacing(10);
+        GridPane.setConstraints(hbox, 0, 0);
+        
+        ImageView aboutIcon = new ImageView();
+        configureIcon(imageIcon, aboutIcon);
+        
+        Text iconText = new Text(PROG_NAME);
+        Font iconFont = new Font(FONT_SIZE * 7);
+        iconText.setFont(iconFont);
+        
+        hbox.getChildren().addAll(aboutIcon, iconText);
+           
+        Font stdFont = new Font(FONT_SIZE);
+        
+        TextFlow introductionFlow = new TextFlow();
+        introductionFlow.getChildren().add(getText("monitoring.aboutdialog.aboutIntroduction", stdFont));
+        GridPane.setConstraints(introductionFlow, 0, 1); 
+        
+        TextFlow osipVersionFlow = new TextFlow();
```

Comment:
Hier, finde ich, kannst du die Version und die Einleitung in einem TextFlow unterbringen.